services/users-api/lambdas/fetchReferenceCode.js

```diff
@@ -0,0 +1,82 @@
+import to from 'await-to-js';
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+
+import config from '../../../config';
+import * as response from '../../../libs/response';
+import * as dynamoDb from '../../../libs/dynamoDb';
+import log from '../../../libs/logs';
+
+/**
+ * Get the referenceCode associated with the user that has the given email or PN
+ */
+export const main = async (event, context) => {
+  const query = event.pathParameters?.query;
+
+  if (!query) {
+    return response.failure({
+      status: 400,
+      message: 'Missing required parameter: "query"',
+    });
+  }
+
+  const isEmail = query.includes('@');
```

Comment:
Checking if the query is an email using `includes` is perhaps not the best solution. Could you use regex instead?